arcgis-ios-sdk-samples.xcodeproj/project.pbxproj

```diff
@@ -4046,23 +4062,23 @@
 			path = "Create terrain surface from a local tile package";
 			sourceTree = "<group>";
 		};
-		F15CF0C723EE26CB0038F052 /* Display subtype feature layer */ = {
+		F140E1E823F35E61008AC44E /* Display annotation */ = {
```

Comment:
Same here. The identifier or existing groups shouldn't change.

Reply:
My guess is that Xcode is doing some self-sanitizing job for the project file altered manually. The changes shown here does not change the content - they are just lines swapped in order.

Will do some extra research on how Xcode manage project file itself.